### PR TITLE
add answers for invalid and not enough info to resolve market 

### PR DESCRIPTION
**Pull request by richardblythman**
See also #186 of mech https://github.com/valory-xyz/mech/pull/186

### CODE REVIEW

File: benchmark/benchmark/run_benchmark.py [hunk 1]
```diff
@@ -32,6 +33,8 @@ def tool_map(tool):
         return prediction_request_sme
     elif tool in ["claude-prediction-offline", "claude-prediction-online"]:
         return prediction_request_claude
+    elif tool in ["prediction-request-rag"]:
```
Comment (Adamantios) [inline benchmark/benchmark/run_benchmark.py:36]:
```suggestion
    elif tool == "prediction-request-rag":
```

However, this whole `tool_map` method could be a mapping instead. Then use `.get(name, None)` and if `None`, raise the Exception below.

File: benchmark/benchmark/run_marketclosing.py [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+        test_q["prediction"] = "yes"
+    elif result.dict().get("has_occurred") == False:
+        test_q["prediction"] = "no"
+    elif result.dict().get("is_valid") == False:
+        test_q["prediction"] = "invalid"
+        return test_q
+    elif result.dict().get("is_determinable") == False:
+        test_q["prediction"] = "indeterminable"
+    else:
+        test_q["prediction"] = None
```
Comment (Adamantios) [inline benchmark/benchmark/run_marketclosing.py:71]: Comparing bools with `==` instead of `is`.

**Review by Adamantios — APPROVED** [review]
Looks good, left some minor comments.

**Comment by richardblythman** [comment]
Thanks @Adamantios.
Updated and should be good to merge.

**Pull request merged**